src/api/calld.js

```diff
@@ -101,14 +101,14 @@ export default (client: ApiRequester, baseUrl: string) => ({
       ApiRequester.successResponseParser,
     ),
 
-  answerSwitchboardHeldCall: (switchboardUuid: UUID, callId: string) =>
-    client.put(`${baseUrl}/switchboards/${switchboardUuid}/calls/held/${callId}/answer`),
+  answerSwitchboardHeldCall: (switchboardUuid: UUID, callId: string, lineId: string) =>
+    client.put(`${baseUrl}/switchboards/${switchboardUuid}/calls/held/${callId}/answer?line_id=${lineId}`),
```

Comment:
This can break the retro-compatibility of the SDK by calling `...?line_id=*empty*` for someone who updated the SDK without updating the call to `answerSwitchboardHeldCall `.

Can you mark the `lineId` params as optional and add the parameter only if `lineId` is set ?

Reply:
good idea